test/validation/api/pktio/pktio.c

```diff
@@ -3316,7 +3316,7 @@ static void pktio_test_pktout_compl(bool use_plain_queue)
 		opt.queue = compl_queue[i];
 		opt.mode = ODP_PACKET_TX_COMPL_ALL;
 		odp_packet_tx_compl_request(pkt_tbl[i], &opt);
-
+		CU_ASSERT(odp_packet_has_tx_compl_request(pkt_tbl[i]) == 1);
```

Comment:
Validation test changes should not be in the same commit with linux-gen implementation changes. It's easier to merge test changes to other implementations, when impl and test changes are separate.

Reply:
Moved to separate commit in v2.